### PR TITLE
Feat/pie chart

### PR DESCRIPTION
**Pull request by Nhlakanipho12**
Screenshot from 2020-08-06 00-35-19

### CODE REVIEW

**Comment by Portia-arch** [comment]
@Nhlakanipho12 please fix the conflicts also

**Comment by Shaheedah** [comment]
Please Review --->
Screenshot from 2020-09-01 09-02-12

**Comment by XolaniMgube** [comment]
Hey.
I couldn't find the #254 ticket so I could refer to it's description, But this still looks good. Easy understand what's going on. The only thing I find odd is the green colour for the red flag part. That could be because you used react-plotly  and it generates the chart automatically, so isn't there a way you can specify the colours for each label? Unless the colours don't really matter, then its fine because it looks good.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This is good.

But I'm seeing some node_module files in your pr. So something weird is happening there.  Rememver that node_modules should never be committed.

Please take a look over your PR yourself and get rid of the extra stuff

**Review by Lebogang-cloud — COMMENTED** [review]
Good work

Your pie chart can accept values as part of the requirements. It is easy to understand.

**Comment by Lisa1400** [comment]
Well done, it looks really good. From my side just make it prettier, the code has open spaces.

**Review by Lisa1400 — APPROVED** [review]

File: experimental-frontend/package.json [hunk 1]
```diff
@@ -6,9 +6,11 @@
     "@material-ui/core": "^4.9.0",
     "@material-ui/icons": "^4.5.1",
     "@material-ui/styles": "^4.9.0",
+    "chart.js": "^2.9.3",
     "material-ui": "^0.20.2",
     "plotly.js": "^1.54.1",
     "react": "^16.11.0",
+    "react-chartjs-2": "^2.10.0",
```
Comment (sheenarbw) [inline experimental-frontend/package.json:13]: What is chartjs for?  Plotly was already installed

File: frontend/src/components/regions/PieChartDemo/Presentation.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import Presentation from "./Presentation";
+
```
Comment (sheenarbw) [inline frontend/src/components/regions/PieChartDemo/Presentation.js:3]: Please remove this file. 

Remember that you are working on th experimental frontend only. What is this for?

File: frontend/src/components/regions/PieChartDemo/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Presentation from "./Presentation";
+
+import Plot from 'react-plotly.js';
+
+class App extends React.Component {
+  render() {
+    return (
```
Comment (sheenarbw) [inline frontend/src/components/regions/PieChartDemo/index.js:8]: Please remove this file. 

Remember that you are working on th experimental frontend only. What is this for?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
There are node_modules and also changes to the main frontend.

Always be careful to only work on code you are meant to work on. Why do you think it would be good to add a pie chart with nonesense data to the main frontend? You know people use that  thing, right? 

Please make sure that no piece of code you are writing here has anything to do with the `frontend` folder. Every change you make should be inside the `experimental-frontend` folder

**Comment by Shaheedah** [comment]
Hi Sheena I think you reviewed the old code, changes were made on the PR from the 27th of August. Please see these commits:
- 60b92cba6ff75898fab54a089bae48c20aef3c05
- fda38dbb737bd326135b1c90c37c183c4e5f38e1
- f1ebc63362bba16d51ce2b1b809db727f1f917ba
- 88e6cc2a2489b24e24d08f710eb942ba68897edd
- e1dac5cc1e7ab8b4f93175ed6126b9d77eefec12
What we still need to do is remove the node modules on the frontend folder

**Pull request closed without merging**